farsetlabs.json

```diff
@@ -0,0 +1,38 @@
+{
+    "api": "0.13",
+    "botname": "FarBot",
```

Comment:
What's this thing? I don't think we've got a bot.

Reply:
we _used_ to, (@andrewbolster ) removed for now :)